developer/customization/articles/02-architecture/03-understanding-services/02-osgi-cdi-integration/02-using-osgi-services-in-a-bean.markdown

```diff
@@ -0,0 +1,37 @@
+# Using OSGi Services in a Bean 
+
+Any bean can use the `@org.osgi.service.cdi.annotations.Reference` annotation to
```

Comment:
in the provide a service use case you don't use the FQCN. I would try to be consistent. Either use FQCN or not :)

Reply:
Used FQCN consistently now. Thanks.